src/main/java/gregtech/api/block/machines/BlockMachine.java

```diff
@@ -177,6 +175,11 @@ public float getExplosionResistance(@Nonnull World world, @Nonnull BlockPos pos,
         return collisionList;
     }
 
+    @Override
+    public boolean doesSideBlockRendering(@Nonnull IBlockState state, @Nonnull IBlockAccess world, @Nonnull BlockPos pos, @Nonnull EnumFacing face) {
+        return state.isOpaqueCube() && getMetaTileEntity(world, pos) != null;
```

Comment:
Should this have an additional check for if the face to be rendered is obscured by another block?

Reply:
this is actually a method called by another block to check side rendering.

Reply:
ah, I see